CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## 0.1.6
+
+- Fix invalid require that pervent the plugin to be registred properly,
```

Comment:
This can be simply "removed require statement for a file that is no longer present in logstash-core"